lnrpc/lightning.proto

```diff
@@ -2567,6 +2567,15 @@ message PendingChannelsResponse {
 
         // Previously used for confirmation_height. Do not reuse.
         reserved 2;
+
+        // The number of blocks until the funding transaction is considered
+        // expired. If this value gets close to zero, there is a risk that the
+        // channel funding will be canceled by the channel responder. The
+        // channel should be fee bumped using CPFP (see walletrpc.BumpFee) to
+        // ensure that the channel confirms in time. Otherwise a force-close
+        // will be necessary if the channel confirms after the funding
+        // transaction expires.
```

Comment:
Add "A negative value means the channel responder has very likely canceled the funding and the channel will never become fully operational."

Reply:
@guggero @yyforyongyu : We should be able to test for a negative value of `FundExpiryBlocks`. Any tips on how to construct this state?  I was considering something like:

1. Alice opens a pending channel to Bob
2. Alice force closes the channel
3. Mine more blocks than what Bob expects for a confirmation
4. Verify the negative value of `FundExpiryBlocks`

I'll try it out, but it's not clear to me how a "pending" channel might transition after the requisite number of confirmations or a force close. Thanks!

Reply:
There is a `ht.MineEmptyBlocks` method you can use, which mines blocks, but not includes any txes from the mempool.

You could try it out locally, and I'm suggesting not adding it to the itest builds here. Unfortunately, we are still trying to find out an efficient way to run itests faster, and mining 2016 blocks will dramatically slow them down.